cinema/urls.py

```diff
@@ -1 +1,25 @@
-# write urls here
+from django.urls import path, include
+
+from rest_framework import routers
+
+from cinema.views import (
+    CinemaHallViewSet,
+    MovieViewSet,
+    MovieSessionViewSet,
+    GenreViewSet,
+    ActorViewSet,
+)
+
+router = routers.DefaultRouter()
+router.register("movies", MovieViewSet)
+router.register("movie_sessions", MovieSessionViewSet)
+router.register("genres", GenreViewSet)
+router.register("actors", ActorViewSet)
+router.register("cinema_halls", CinemaHallViewSet)
+
+urlpatterns = [
+    path("", include(router.urls)),
+
+]
```

Comment:
No need to include here, because base path not provided. SImplify this:
urlpatterns = router.urls